selfdrive/controls/controlsd.py

```diff
@@ -94,7 +94,6 @@ def __init__(self, sm=None, pm=None, can_sock=None):
     cp_bytes = self.CP.to_bytes()
     params.put("CarParams", cp_bytes)
     put_nonblocking("CarParamsCache", cp_bytes)
-    put_nonblocking("LongitudinalControl", "1" if self.CP.openpilotLongitudinalControl else "0")
```

Comment:
Should we remove the param from params.py. As far as I can tell it's unused now.

Reply:
done